### PR TITLE
Fix Cross Evo Crash

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

**Comment by MacChaeger** [comment]
3500!

**Comment by Spandamn** [comment]
C H E C K E M
H
E
C
K
E
M

File: config/formats.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ exports.Formats = [
 			return this.validateSet(set, teamHas, mixedTemplate);
 		},
 		onModifyTemplate: function (template, pokemon) {
-			if (pokemon.crossEvolved || !pokemon.set.name) return template;
```
Comment (acz13) [inline config/formats.js]: is removing `!pokemon.set.name` safe?

**Comment by panpawn** [comment]
Can we do more specific commit names :|

**Comment by Zarel** [comment]
oh wow I thought it would take the pullreq name...

**Pull request merged**